### ISSUE TITLE
Failing gracefully

### ISSUE THREAD

**Issue by Ericxgao** [enigmampc/catalyst#202]
Is there a good way to fail gracefully - e.g. if I don't have enough balance, just ignore the error and continue with the minute callbacks without needing to completely restart the algorithm?

**Comment by fredfortier** [comment]
You can try ... except anything inside of your handle_data method, but
things like not enough balance are raised outside so you may not be able to
catch it. We can add an option to ignore these types of errors and warn
instead.

**Comment by Ericxgao** [comment]
Yeah, unfortunately I have to restart the algo if it fails outside of that handler. Would be a nice option to have a warning - thanks.
